Simulator/components/App.jsx

```diff
@@ -92,6 +97,16 @@ class App extends Component {
     const hasData = !!this.props.tables && Object.keys(this.props.tables).length > 0;
     const isAddressBarEmpty = (this.props.addressBarUrll === '');
 
+    const hasStandardConnections = this.props.standardConnections.length > 0;
+    if (hasStandardConnections) {
+      const connectionList = this.props.standardConnections;
+      standardConnectionsWindow = connectionList.map((standardConnection, idx) =>
```

Comment:
(nitpick) I could be wrong but, this seems like a list of tabs, not a window so the name might be confusing